.osu-wiki-bin.json

```diff
@@ -0,0 +1,4 @@
+{
+  "osuApiKey": "",
```

Comment:
you accidentally committed this (perhaps we should add it to `.gitignore`)

Reply:
ah my bad. so i have to delete this file, right?